### ISSUE TITLE
Add Accessibility Testing to Design Tokens (NICE TO HAVE - Non critical)

### ISSUE THREAD

**Issue by garrettbear** [MetaMask/design-tokens#657]
**Title:** Add Accessibility Testing to Design Tokens

### Description
To ensure our design tokens meet accessibility standards, this ticket focuses on integrating accessibility testing into our design token workflow. This will help identify and rectify any accessibility issues early in the design process, particularly focusing on color contrast, typography readability, and overall compliance with WCAG guidelines.

### Technical Details
- Implement automated accessibility testing tools that can analyze the design tokens for compliance with WCAG 2.1 guidelines.
- Set up tests for color contrast, font sizes, and spacing to ensure they meet accessibility standards.
- Integrate these tests into our continuous integration (CI) pipeline to automatically run with every update to the design tokens.

### Acceptance Criteria
1. Automated accessibility testing with A11Y storybook plugin
2. Created stories with a comprehensive text and background color combinations (and varying text size?, if not picked up automatically) - Call out every combination of text and background color. NICE TO HAVE: Add the contrast ratios for each combination
3. Document what text/color combinations are not accessible - Any design tokens failing the accessibility tests are identified and documented.
4. See attached screenshot for expected output.

**Comment by georgewrmarshall** [comment]
![Screenshot 2024-04-15 at 9 56 18 AM](https://github.com/MetaMask/design-tokens/assets/8112138/4803c121-d0c7-452f-8ca1-b85b1b2286d0)
